playbooks/ci-full-centos/tasks/pre-deploy.yml

```diff
@@ -0,0 +1,98 @@
+---
+- name: controller 0 hosts file
+  hosts: controller[0]
+  vars_files:
+  - ../vars/main.yml
+  tasks:
+    - name: add controller node to /etc/hosts
+      lineinfile: dest=/etc/hosts regexp={{ testenv_instance_names[0] }}
+                  insertafter=EOF
+                  line="127.0.1.1 {{ testenv_instance_names[0] }}"
+
+- name: controller 1 hosts file
+  hosts: controller[1]
+  vars_files:
+  - ../vars/main.yml
+  tasks:
+    - name: add controller node to /etc/hosts
+      lineinfile: dest=/etc/hosts regexp={{ testenv_instance_names[1] }}
+                  insertafter=EOF
+                  line="127.0.1.1 {{ testenv_instance_names[1] }}"
+
+- name: compute 0 hosts file
+  hosts: compute[0]
+  vars_files:
+  - ../vars/main.yml
+  tasks:
+    - name: add compute node to /etc/hosts
+      lineinfile: dest=/etc/hosts regexp={{ testenv_instance_names[2] }}
+                  insertafter=EOF
+                  line="127.0.1.1 {{ testenv_instance_names[2] }}"
+
+- name: tasks for all
+  hosts: all
+  serial: 10
+  tasks:
+  # Set Locale
+  - name: Generate en_US locale
+    command: localedef -c -f UTF-8 -i en_US en_US.UTF-8
```

Comment:
Not sure if this is enough to achieve the Debian-equivalent (e.g. do I need to export LC_ALL in an Ansible-y way)... does this even matter for tests? I don't actually know.